mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -192,8 +192,9 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7350 resetLastAttackedTicks ()V
 	METHOD method_7351 getBlockBreakingSpeed (Lnet/minecraft/class_2680;)F
 		ARG 1 block
-	METHOD method_7353 addChatMessage (Lnet/minecraft/class_2561;Z)V
+	METHOD method_7353 addGameMessage (Lnet/minecraft/class_2561;Z)V
```

Comment:
Hmm, `addMessage` would also work here.

Reply:
isn't it too broad?

Reply:
`Game` doesn't really tell you anything since everything in Minecraft is related to a game.